### ISSUE TITLE
Move latLonToXYZ to utils?

### ISSUE THREAD

**Issue by vorg** [variablestudio/pex-cam#6]
Currently they live in orbiter

```
module.exports.latLonToXyz = latLonToXyz
module.exports.xyzToLatLon = xyzToLatLon
```

**Comment by vorg** [comment]
Use those modules instead
https://www.npmjs.com/package/xyz-to-latlon
https://www.npmjs.com/package/latlon-to-xyz

**Comment by vorg** [comment]
Fixed in 61c34295923a32dcaaa5cf1ad7e26ff0d48ff413
